scripts/jenkins/qat/Makefile

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Makefile for QAT-DPDK jenkins.
+
+all: plugin tests
+
+plugin:
+	bash plugin-deploy.sh
+
+tests:
+	bash tc-deploy.sh
```

Comment:
The scripts should either have `/bin/bash` in their shebangs or run with `/bin/sh` in this Makefile for consistency to avoid future confusions caused by unnoticed bashisms in the scripts.

Reply:
done :)